### DISCUSSION TITLE
CS50x

### DISCUSSION THREAD

**Discussion by ABUKHALED3** [community/community#36144] [Codespaces] [answered]
![Capture PNG114](https://user-images.githubusercontent.com/113803524/195869445-25abc185-36f2-41cd-a9cf-f50b9f7a5797.PNG)

Dir,
From 7 am until now Codespaces is not working for me and this shows in every reload window, I hope it will be fixed soon.

**Comment by thomas-sickert** [comment, marked as answer]
Sorry you ran into issues. Were you able to get reconnected?

It doesn't look like this happened during a recent incident, but if you run into this again, it may be helpful to check our [status page](https://www.githubstatus.com/) or the [@githubstatus](https://twitter.com/githubstatus) twitter account to see the status of the service and get updates on fixes. If you don't see anything there, feel free to reach out to [support](https://support.github.com/request)
